### ISSUE TITLE
Make sure iter8 shows up on top in google searches

### ISSUE THREAD

**Issue by sriumcp** [iter8-tools/docs#300]

**Comment by sriumcp** [comment]
Google has documentation on this.

**Comment by Alan-Cha** [comment]
@fabolive [https://github.com/istio-ecosystem/iter8](https://github.com/istio-ecosystem/iter8) shows up right below [iter8.tools](iter8.tools), but there is nothing important inside the repository. Is it possible to delete it?

**Comment by Alan-Cha** [comment]
Something we can do is to create a sitemap which can allow Google and other search engines to better understand our site. However, I think it might be unnecessary. According to [https://support.google.com/webmasters/answer/156184](https://support.google.com/webmasters/answer/156184), we might not need a sitemap if our site is less than 500 pages. I do wonder if we should do it for the archival site and the blog though.

**Comment by Alan-Cha** [comment]
I finished reading this [article](https://support.google.com/webmasters/answer/7451184?hl=en), which is Google's official documentation on Search Engine Optimization. It seems to me that most of what's described is about creating meaningful and easily navigable websites. There's is nothing concrete that we can plug into to raise our ranking.

However, the last time I checked, we are already the first entry under "iter8". I think for the time being, it makes sense to close this issue.
